fe-library-app/src/components/AddModifyForm/AddModifyBookForm.tsx

```diff
@@ -0,0 +1,288 @@
+import { ChangeEvent, useCallback, useEffect, useState } from 'react'
+
+import Select, { MultiValue } from 'react-select'
+
+import AddAuthorModal from '../../modals/AddAuthorModal'
+import { GetAuthorResponse, getAuthors } from '../../services/AuthorService'
+import { createBook, getBookById, modifyBook } from '../../services/BookService'
+import AddAuthor from '../AddAuthor/AddAuthor'
+import DefaultBookCover from '../BookList/BookCard/DefaultBookCover.png'
+import styles from './AddModifyBookForm.module.css'
+
+interface AddModifyBookFormProps {
+  onFinish: () => void,
+  onHide?: () => void,
+  id?: string
+}
+
+const AddModifyBookForm = (props: AddModifyBookFormProps) => {
+  const [ authorList, setAuthorList ] = useState<GetAuthorResponse[]>([])
+  const [ selectedAuthors, setSelectedAuthors ] = useState<GetAuthorResponse[]>([])
+  const [ newBookTitle, setNewBookTitle ] = useState('')
```

Comment:
Why did you separate new book properties instead of having one object that stores all the info?

Reply:
At that moment It was confusing for me to handle it. I'm gonna change that now